App/Resources/English.lproj/MainMenu.nib/designable.nib

```diff
@@ -273,6 +273,11 @@ CA
                                     <action selector="showFontFileName:" target="-1" id="Ppi-rz-b6J"/>
                                 </connections>
                             </menuItem>
+                            <menuItem title="Show Font Metrics" keyEquivalent="6" id="Xhr-L4-CtF">
```

Comment:
The menu still shows `Show Metrics`, is there somewhere else I need to update?

Reply:
If I open `App/Resources/English.lproj/MainMenu.nib` in XCode and save it, the menu entries is updated, but it changes `App/Resources/English.lproj/MainMenu.nib/designable.nib`, `App/Resources/English.lproj/MainMenu.nib/keyedobjects.nib`, and create a new `App/Resources/English.lproj/MainMenu.nib/keyedobjects-101300.nib` file. Should I commit all of these?